rust/crd/src/affinity.rs

```diff
@@ -73,6 +73,9 @@ mod test {
                 preferred_during_scheduling_ignored_during_execution: Some(vec![
                     WeightedPodAffinityTerm {
                         pod_affinity_term: PodAffinityTerm {
+                            mismatch_label_keys: None,
+                            match_label_keys: None,
+
```

Comment:
suggestion: Add these new fields at the end of the struct instead of at the start. Also, we should a comment like "Both these fields were added in Kubernetes 1.30, and cannot be used for now.".

Reply:
done